buildLogAnalyzer.py

```diff
@@ -410,6 +411,11 @@ def makeHTMLLogFile(self, pkg):
             )  # do this first to not escape it again in the next subs
             newLine = newLine.replace("<", "&lt;").replace(">", "&gt;")
             if lineNo in pkg.errLines.keys():
+                m = linePartsUrl.match(newLine)
+                if m:
+                    branch = os.getenv("CMSSW_VERSION", "master")
```

Comment:
@iarspider , instead of getting env for every log line. Update https://github.com/cms-sw/cms-bot/blob/master/buildLogAnalyzer.py#L773 e.g. change it to `rel = os.getenv("CMSSW_VERSION", "master")` and then use `self.release` instead of `branch`

Reply:
@iarspider , please also update  https://github.com/cms-sw/cms-bot/blob/master/buildLogAnalyzer.py#L773  to use `master` if env is not set